### PR TITLE
#113 ui

### PR DESCRIPTION
**Pull request by mgpaja8**
Reopen incidents

### CODE REVIEW

**Comment by j0nscalet** [comment]
@mgpaja8 I noticed that IncidentTests 'Cant_Comment_On_Resolved_Incidents' fails. Since we discussed removing that business rule I was wondering if you've got that logic in another branch?

If not I can just remove the rule, let me know what you think.
Thanks

**Comment by j0nscalet** [comment]
@mgpaja8 sounds like we just need to delete the test since the BusinessRule is deleted.

File: src/QualityKitchen.Domain/Model/Incident/Incident.cs [hunk 1]
```diff
@@ -161,8 +159,28 @@ public Activity Resolve(string employeeCode, string managerName, bool isManager,
...
+        public Activity Reopen(string employeeCode, string managerName, bool isManager, string comment)
+        {
+            ThrowIf.NullOrEmpty(employeeCode, nameof(employeeCode));
+            ThrowIf.NullOrEmpty(managerName, nameof(managerName));
+            ThrowIf.NullOrEmpty(comment, nameof(comment));
+
+            BusinessRules.OnlyManagersCanResolveAndReopenIncidents(isManager);
+            BusinessRules.OnlyResolvedIncidentsCanBeReopened(this.Status);
+
+            this.MakeComment(employeeCode, managerName, isManager, comment);
```
Comment (j0nscalet) [inline src/QualityKitchen.Domain/Model/Incident/Incident.cs]: Comment is not required to re-open an incident. Sure a manager could include a comment if they wanted, but it's not required.

Comment (mgpaja8) [inline src/QualityKitchen.Domain/Model/Incident/Incident.cs]: So I should allow user to send null as a comment, and then just if comment != null V empty, I made comment?

**Pull request closed without merging**